web/src/components/TimeSlider.tsx

```diff
@@ -93,10 +100,11 @@ export function TimeSliderBasic({
       </SliderPrimitive.Track>
       <SliderPrimitive.Thumb
         data-test-id="time-slider-input"
-        className={`block h-6 w-6 rounded-full bg-white
-          bg-center bg-no-repeat shadow-3xl focus:outline-none
-          focus-visible:ring focus-visible:ring-brand-green/10
-          focus-visible:ring-opacity-75 dark:bg-gray-400 dark:focus-visible:ring-white/70`}
+        className={`block h-6 w-6 rounded-full bg-white bg-center
+          bg-no-repeat shadow-3xl hover:ring
```

Comment:
How about a bit of transition to make it more smooth? 🧈 
```suggestion
          bg-no-repeat shadow-3xl transition-shadow hover:ring
```

Reply:
Let's do it!